_posts/2020-07-30-image-redux.md

```diff
@@ -0,0 +1,474 @@
+---
```

Comment:
Is it possible this file `_posts/2020-07-30-image-redux.md ` has sneaked in here by mistake? I don't think it should get lumped in with this pull request.

Reply:
Yeah it looks like this got added in commit ( https://github.com/dask/dask-blog/pull/77/commits/86ed738578f5f3981990759d1acbadd62003ad34 ). @quasiben is it ok if we just delete this file or is there some content that should be moved to `_posts/2020-10-23-deconvolution.md`?

Reply:
This has since been removed